modules/id5IdSystem.js

```diff
@@ -303,6 +386,32 @@ class IdFetchFlow {
     }
     return data;
   }
+
+  // eslint-disable-next-line no-dupe-class-members
+  #processFetchCallResponse(fetchCallResponse) {
+    try {
+      resetNb(this.submoduleConfig.params.partner);
+      if (fetchCallResponse.privacy) {
+        storeInLocalStorage(ID5_PRIVACY_STORAGE_NAME, JSON.stringify(fetchCallResponse.privacy), NB_EXP_DAYS);
+      }
+    } catch (error) {
+      logError(LOG_PREFIX + error);
+    }
+    return fetchCallResponse;
+  }
+
+  // eslint-disable-next-line no-dupe-class-members
+  async #loadExternalModule() {
+    const url = this.submoduleConfig.params.externalModuleUrl;
+    return new GreedyPromise((resolve, reject) => {
```

Comment:
as far as I know this must be done through our [centralized utility](https://github.com/prebid/Prebid.js/blob/master/src/adloader.js#L43)

Reply:
We thought that since the publisher is in charge to enable this by putting the script URL in the configuration, we are not subject to the policy because the loading is only optionally triggered and not by default.
The problem with the policy associated to the centralized utility is that we are providing extra functionality to our user id module which has been developed within our own open source javascript integration library which can be found at https://github.com/id5io/id5-api.js and for us would be quite difficult to have a change in ownership of the repository and build process to comply to the policy.

Reply:
you cannot call appendChild, you must use the adloader utility

Reply:
I am not sure I follow your proposal, it being open source or decoupled seems to have nothing to do with if you use our utility or not to load it.

Reply:
My assumption was that the policy was a prerequisite for using the utility. I applied changes to make use of the mentioned utility and it's working fine although the utility doesn't handle gracefully errors while loading the external scripts since it will never call the callback but limit itself to log an error in the console. This makes it impossible to use a fallback logic in case the browser wasn't able to load the script.

Reply:
@dgirardi are we good to go here now that we're using the utility as requested?